rune/libenclave/internal/runtime/pal/skeleton/sgxsign.c

```diff
@@ -91,6 +104,21 @@ static RSA *load_sign_key(const char *path)
 		return NULL;
 	}
 
+	const BIGNUM *e = get_exponent(key);
+	char *p = BN_bn2dec(e);
+	if (memcmp(p, "3", 2)) {
+		fprintf(stderr, "Exponent must be set to 3.\n");
+		RSA_free(key);
+		return NULL;
+	}
+
+	if (BN_num_bytes(e) != 1) {
+		fprintf(stderr, "Invalid exponent size %d\n",
+			BN_num_bytes(get_exponent(key)));
+		RSA_free(key);
+		return NULL;
+	}
+
```

Comment:
use `BN_is_word(e, 3)` instead to simplify the code.

Reply:
Done.